flake.nix

```diff
@@ -1,7 +1,7 @@
 {
   description = "agenix-cli";
 
-  inputs.nixpkgs.url = "github:nixos/nixpkgs/nixos-20.09";
+  inputs.nixpkgs.url = "github:nixos/nixpkgs/release-21.11";
```

Comment:
The release branch is what commits get merged into -- so it's basically the "master" branch of the 21.11 release, which is not reallly what we want.

```suggestion
  inputs.nixpkgs.url = "github:nixos/nixpkgs/nixos-21.11";
```

Reply:
Ah, thanks! I didn't realize that.